include/swift/AST/DiagnosticsClangImporter.def

```diff
@@ -160,77 +160,76 @@ NOTE(incomplete_interface, none, "interface %0 is incomplete", (const clang::Nam
 NOTE(incomplete_protocol, none, "protocol %0 is incomplete", (const clang::NamedDecl*))
 NOTE(incomplete_record, none, "record '%0' is not defined (incomplete)", (StringRef))
 NOTE(record_over_aligned, none, "record '%0' is over aligned", (StringRef))
-NOTE(record_non_trivial_copy_destroy, none, "record '%0' is not trivial to copy/destroy", (StringRef))
+NOTE(record_non_trivial_copy_destroy, none, "record '%0' is not trivial to copy or destroy", (StringRef))
 NOTE(record_is_dependent, none, "record '%0' is dependent", (StringRef))
 NOTE(record_parent_unimportable, none, "record %0's parent is not importable", (StringRef))
 NOTE(reference_passed_by_value, none, "function uses foreign reference type "
                                       "'%0' as a value in %1 types which breaks "
                                       "'swift_shared_reference' contract (outlined in "
-                                      "C++ Interop User Manual).",
+                                      "the C++ interoperability user manual)",
```

Comment:
Is this something we really want to be pointing people at?

Reply:
We should remove this reference to User manual. This would be become stale once we have documentation elsewhere.

Reply:
Yup, removing it